mesos-tools/src/cgcloud/mesos_tools/__init__.py

```diff
@@ -22,14 +22,19 @@
 
 log = logging.getLogger( __name__ )
 
+shared_dir='/home/mesosbox/shared/'
 
-class MesosTools( object ):
+work_dirs = ['/var/lib/mesos', '/var/lib/toil']
 
```

Comment:
The `mesos` subproject can't reference anything Toil-specific.

I have the feeling both shared_dir and work_dirs should be parameters to the MesosTools.

When defining a constant, there should only be one definitions, not two or more that have to be kept in sync. As a very last resort (not here though) all definitions have to be annotate with a comment that clearly states where the other definitions are and that they need to be kept in sync.